draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -952,6 +952,30 @@ In contrast, this document is concerned with encodings from arbitrary bit string
 to elliptic curve points.
 This document does not cover serialization or deserialization.
 
+### Domain separation {#term-domain-separation}
+
+In most cases, cryptographic protocols that use random oracles are analyzed
+under the assumption that the random oracle answers only queries generated
+by that protocol.
+In practice, this assumption may not hold: commonly, two or more protocols
+may model the same hash function as a random oracle, which violates the above
+assumption if both protocols compute the hash of the same value.
```

Comment:
Perhaps add:

```
That is, let R() be a random oracle used by protocols P1 and P2. If P1 and P2 ever query R with the same value x, the assumption above is violated.
```

And maybe then show how P1 and P2 would use R1 and R2 (as defined below) to address this?

Reply:
Yeah, I think the first paragraph is unclear in a couple ways. I'll try to address and incorporate the above.

Reply:
Done, I think.